src/controllers/admin/UserAjax.php

```diff
@@ -329,12 +335,15 @@ public function getRetrieve()
 
             $userModel = new User();
             $user = $userModel->getById($params->id);
+            if(empty($user)){
+                throw new \Exception('User not found.');
+            }
             $output = array('id'       => $user->getId(),
-                              'email'    => $user->getEmail(),
-                              'role'     => $user->getRole(),
-                              'name'     => $user->getName(),
-                              'last_login' => $user->getLastLogin(),
-                              'gateway_customer_id'=> $user->getGatewayCustomerId()
+                            'email'    => $user->getEmail(),
+                            'role'     => $user->getRole(),
```

Comment:
we should have this return an object or array including the Role ID and the user friendly Role Name string.